test/unit/warpable_test.rb

```diff
@@ -49,16 +49,4 @@ def setup
     assert_not_nil @warp.user_id
     assert_equal @warp.map.user_id, @warp.user_id
   end
-
-  test "try export" do
```

Comment:
Just a small observation. I guess we should try to make a standard for test naming, and use only either single or double quotes :)

Reply:
I'm kinda trying to solve those code pattern issues by enforcing a style guide(rubocop) in #547

Reply:
Sounds great, thank you!